mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreRightManager.java

```diff
@@ -74,6 +74,10 @@ public StoreRightManager(MailboxSessionMapperFactory mailboxSessionMapperFactory
         this.eventBus = eventBus;
     }
 
+    private boolean isCrossDomainAccessAllowed() {
+        return Boolean.parseBoolean(System.getProperty("james.rights.users.crossdomain", "false"));
```

Comment:
Make it a constant: looking this up at class loading is enough

Reply:
Ah ok I got it, that's needed because of the tests!

My suggestion is to allow the constant to be package local mutable annotated with `@VisibleForTesting` so that overloading that in tests become easy while keeping the property lookup at classloading time.

WDYT?